### PR TITLE
Implementation of Unique Durable Identifier for Volume

### PR DESCRIPTION
**Pull request by NajmudheenCT**
**What this PR does / why we need it**:
This is the implementation of model changes required for Unique Durable Identifier for Volume as mentioned in  https://github.com/opensds/design-specs/pull/47 .
This aslo includes the LVM driver implementaion.
As part of https://github.com/opensds/opensds/issues/997 , we need to add durable identifier filed in the volume to recognize backend LUN from Vmware platform. 

for  LVM, identifier  is the scsi_id created for iscsi tagets. In this PR we have changes in lvm driver to create scsi_id from volumeid . This scsi_id will be used to create targets and the same will be used as identifier. 

**Special notes for your reviewer**:

scsi_id is created from volID, whic is UUID generated dynamically.

### CODE REVIEW

File: contrib/drivers/lvm/lvm.go [hunk 1]
```diff
@@ -48,8 +48,9 @@ const (
 )
 
 const (
-	KLvPath  = "lvPath"
-	KLvsPath = "lvsPath"
+	KLvPath     = "lvPath"
+	KLvsPath    = "lvsPath"
+	KLvIdFormat = "NAA"
```
Comment (kumarashit) [inline contrib/drivers/lvm/lvm.go]: Is this fine to keep it all uppercase? Vmware id is in all lowercase naa*

Comment (NajmudheenCT) [reply contrib/drivers/lvm/lvm.go]: Vmware append "naa" for their disk created form a NAA id. keeing "naa" in unique identifer is not required. As per swordfish durableidentifier is an enum , which has "NAA" in caps.
a sample response of swordfish API below. 
"Identifiers": [
{
"DurableNameFormat": "NAA",
"DurableName": "65456765456761001234076100123487"
}
],
We are also targeting the same format.

Comment (kumarashit) [reply contrib/drivers/lvm/lvm.go]: Ok

**Comment by PravinRanjan10** [comment]
LGTM

**Pull request merged**